docs/docs-content/clusters/pcg/deploy-pcg/vmware.md

```diff
@@ -10,6 +10,13 @@ tags: ["pcg"]
 This guide provides you with the steps to deploy a PCG cluster to a VMware vSphere environment. Before you begin the
 installation, carefully review the [Prerequisites](#prerequisites) section.
 
+:::further
+
+For detailed guidance on how to deploy app workloads with a PCG, refer to our
```

Comment:
IMHO this sentence sound nicer the other way around. Like "Refer to our bla bla for detailed guidance on how to bla bla"